backend-zs/models/story.model.js

```diff
@@ -12,13 +12,14 @@ var Story = mongoose.Schema({
     authorImage: {
         type: String,
         required: false,
+        default: ""
     },
     shortDescription: {
         type: String,
         required: true
     },
     story: {
```

Comment:
```suggestion
    storyPages: {
```
This could be more informative, `story` is a little bit confusing.